lib/rust/ensogl/component/toggle-button/src/lib.rs

```diff
@@ -202,27 +204,28 @@ pub struct ToggleButton<S: Shape> {
     model:   Rc<Model<S>>,
 }
 
-impl<Shape: ColorableShape + 'static> Default for ToggleButton<Shape> {
-    fn default() -> Self {
-        let frp = Frp::new();
-        let model = Rc::new(Model::<Shape>::new());
-        Self { frp, model }.init_frp()
-    }
-}
-
 impl<Shape: ColorableShape + 'static> ToggleButton<Shape> {
     /// Constructor.
-    pub fn new() -> Self {
-        default()
+    pub fn new(app: &Application, tooltip_style: tooltip::Style) -> Self {
+        let frp = Frp::new();
+        let model = Rc::new(Model::<Shape>::new());
+        Self { frp, model }.init_frp(app, tooltip_style)
     }
 
-    fn init_frp(self) -> Self {
+    fn init_frp(self, app: &Application, tooltip_style: tooltip::Style) -> Self {
         let network = &self.frp.network;
         let frp = &self.frp;
         let model = &self.model;
         let color = color::Animation::new(network);
         let icon = &model.icon.events;
 
+        // Explicitly define the tooltip placement if the default was used, to ensure that, when
+        // multiple tooltips are being used, we always override to the one we want.
```

Comment:
I don't understand this sentence. What is "the one that we want"? Also, it has some minor English grammar mistakes. See granary or chatgpt to check correctness, please.

Reply:
I've added some extra context. Let me know if you're still confused. Happy to jump into a call to explain it.